src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const listPopulation = document.querySelectorAll('.population');
```

Comment:
there will be only one main list with this class
we can use simple `querySelector` instead of getting the collection of some elements